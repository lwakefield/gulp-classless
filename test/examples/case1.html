<div class="foo bar">
    <div class="foo" class="baz">
        node1
    </div>
</div>
