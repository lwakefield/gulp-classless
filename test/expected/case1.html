<div>
    <div>
        node1
    </div>
</div>
